lab_5_scrapper/scrapper.py

```diff
@@ -176,17 +316,37 @@ def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> No
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        author = article_soup.find('a', class_="article__author-text-link")
+        if author:
+            self.article.author = [author.text.strip()]
+        else:
+            self.article.author = ["NOT FOUND"]
+
+        title = article_soup.find(itemprop="headline")
+        if title:
+            self.article.title = title.text.strip()
+
+        date = article_soup.find('time', class_='meta__text')
+        if date:
+            article_date = date.text
+            formatted_date = ''.join(article_date.split(' в '))
+            self.article.date = self.unify_date_format(formatted_date)
+
+        tags = article_soup.find_all('a', class_='article__tag-item')
+        for tag in tags:
+            self.article.topics.append(tag.text)
 
     def unify_date_format(self, date_str: str) -> datetime.datetime:
         """
-        Unify date format.
+         Unify date format.
 
-        Args:
-            date_str (str): Date in text format
+         Args:
+             date_str (str): Date in text format
 
-        Returns:
-            datetime.datetime: Datetime object
-        """
+         Returns:
+             datetime.datetime: Datetime object
+         """
```

Comment:
please, discard any changes in docstrings

Reply:
fixed